interface/patient_tracker/patient_tracker.php

```diff
@@ -10,71 +10,71 @@
  * @link    http://www.open-emr.org
  * @author  Terry Hill <terry@lilysystems.com>
```

Comment:
TODO
Also plan heavy testing to ensure working right. For example, recurrent appt's show up twice (this will be an easy fix which we had to do when tracker board was initially built).